index.d.ts

```diff
@@ -1,8 +1,47 @@
+declare namespace decamelize {
+	interface Options {
+		/**
+		The separator to use to put in between the words from `string`.
```

Comment:
I think this sentence could be better written. It should also be in the readme.

Reply:
I gave it try. The current sentence was just a copy&paste from the existing docs.